ansible/library/exabgp.py

```diff
@@ -55,7 +55,7 @@
     try:
         line = stdin.readline()
         obj = json.loads(line)
-        f = open("/tmp/exabgp-" + obj["neighbor"]["ip"], "a")
+        f = open("/tmp/exabgp-" + obj["neighbor"]["address"]["local"], "a")
```

Comment:
Why make this change?

Reply:
Because DUT uses loopback address as router ID for both sessions, those two `exabgp` will dump to the same file.